src/frontend/components/VideoPlayer/VideoPlayer.css

```diff
@@ -0,0 +1,59 @@
+.video__player {
+    width: 100%;
+}
+
+.video__title {
+    padding: 2rem 0;
+}
+
+.video__options {
+    align-items: center;
+    justify-content: space-between;
+}
+
+.video__btns {
+    align-items: center;
+    gap: 2rem;
```

Comment:
gap is not supported in some old browsers maybe try achieving this with margins and flex